kermit-io/src/commonMain/kotlin/co/touchlab/kermit/io/RollingFileLogWriter.kt

```diff
@@ -104,11 +104,9 @@ open class RollingFileLogWriter(
   private fun formatMessage(severity: Severity, tag: Tag?, message: Message): String =
     messageStringFormatter.formatMessage(severity, if (config.logTag) tag else null, message)
 
-  private fun maybeRollLogs(size: Long): Boolean {
-    return if (size > config.rollOnSize) {
-      rollLogs()
-      true
-    } else false
+  private fun shouldRollLogs(logFilePath: Path): Boolean {
```

Comment:
I changed the signature of this function because calling `maybeRollLogs(fileSizeOrZero())` seemed cumbersome.